documentation/API_documentation/NumberVerification-Readiness-Checklist.md

```diff
@@ -0,0 +1,25 @@
+# API Readiness Checklist
+
+Checklist for number_verification 1.0.0-rc.1 in release r1.1
+
+| Nr | API release assets  | alpha | release-candidate |  public-release<br>initial | public-release<br> stable | Status | Comments |
+|----|----------------------------------------------|:-----:|:-----------------:|:-------:|:------:|:----:|:----:|
+|  1 | API definition                               |   M   |         M         |    M    |    M   |   Y   | [link](/code/API_definitions/number_verification.yaml) |
+|  2 | Design guidelines from Commonalities applied |   O   |         M         |    M    |    M   |  Y    |      |
+|  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |   Y   |      |
+|  4 | API versioning convention applied            |   M   |         M         |    M    |    M   |   Y   |      |
+|  5 | API documentation                            |   M   |         M         |    M    |    M   |   Y   | Embed documentation into API spec - [link](/code/API_definitions/number_verification.yaml)  |
+|  6 | User stories                                 |   O   |         O         |    O    |    M   |   N   | [link](documentation/API_documentation/) |

```

Comment:
As NumberVerification is aiming for "stable" the User stories need to be marked as "tbd", not "N". User Stories are currently in work with #129 
```suggestion
|  6 | User stories                                 |   O   |         O         |    O    |    M   |   tbd   | [PR #129](https://github.com/camaraproject/NumberVerification/pull/129/files) |
```